samples/edge-inferencing-and-mlops/.pipelines/cd-ml-model.yml

```diff
@@ -0,0 +1,79 @@
+trigger: none
+
+pr: none
+
+resources:
+  pipelines:
+    - pipeline: ci-model
+      source: '[CI] Model'
+      trigger:
+        branches:
+          include:
+            - refs/heads/main
+
+variables:
+  - template: ./variables/dev.yml
+  - group: dev_env_secrets
+  - name: mlModelArtifact
+    value: 'ml-model-files'
+  - name: runAmlModule
+    value: 'train_model'
```

Comment:
Every time CD is run, a new AML pipeline will be registered, correct? Each AML pipeline is a REST endpoint. You don't have to change it, but it'll be nice to update the version of the pipeline rather than creating a new pipeline.

Reply:
This is correct. Every run a new AML pipeline will be registered.